docs/src/configuring-zed.md

```diff
@@ -539,12 +539,6 @@ List of `string` values
 - Setting: `selection_highlight`
 - Default: `true`
 
-## LSP Highlight Debounce
-
-- Description: The debounce delay before querying highlights from the language server based on the current cursor location.
-- Setting: `lsp_highlight_debounce`
-- Default: `75`
-
```

Comment:
I choice this one to remove, as the other one (now line 1330) was more recently added (https://github.com/zed-industries/zed/pull/25974), slightly more detailed, and adjacent to what I am guessing are slightly-more-relevant settings. Happy to switch it if that's better.